kubernetes/remote.yaml

```diff
@@ -121,3 +121,38 @@ spec:
               value: ${STOREBASE}
             - name: CONTAINER_ENV
               value: kubernetes
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: job-creator
+  namespace: default
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: default
+  namespace: default
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: default
+subjects:
+- kind: ServiceAccount
+  name: default
```

Comment:
It's not great that we give `default` a `RoleBinding` but since this is just a minikube dev environment, I suppose it's okay.